css/app.css

```diff
@@ -215,6 +215,18 @@ html, body {
   padding-left: 3px;
 }
 
+#graphiql-container .tool-button {
+  cursor: pointer;
+  text-decoration: none;
+  font-family: sans-serif;
+  color: #ccc;
+  padding: 2px 4px;
+  margin: 0 4px;
+}
+#graphiql-container .tool-button:hover {
+  color: #333;
```

Comment:
Following up on GraphiQL.js@357, maybe the prettify button could change color if it's possible without throwing errors, not on hover?
Also, nit: `.prettify-button`